src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java

```diff
@@ -16,6 +17,8 @@ private Constants() {
     public static final DateTimeFormatter NEW_TIME_PATTERN = DateTimeFormatter.ofPattern("HH:mm");
     public static final DateTimeFormatter DATE_TIME_USER_FRIENDLY_PATTERN =
         DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss");
+    public static final String EW_ENGLAND_AND_WALES = "ET_EnglandWales";
+    public static final String EW_SCOTLAND = "ET_Scotland";
```

Comment:
Just a minor point, could these be ET instead as it makes more sense than EW
```suggestion
    public static final String ET_ENGLAND_AND_WALES = "ET_EnglandWales";
    public static final String ET_SCOTLAND = "ET_Scotland";
```

Reply:
would require updating in the other change as well